Update to 26.1 (#324)
neoforged/Documentation: docs/rendering/screens.md

### Rendering the Screen

> Screens submit their elements for rendering through `#extractRenderStateWithTooltipAndSubtitles` in three different strata: the background stratum, the element stratum, and the optional hoverable stratums.

Reviewer: ```suggestion
Screens submit their elements for rendering through `#extractRenderStateWithTooltipAndSubtitles` in three different strata: the background stratum, the element stratum, and the optional hoverable strata.
```

The background stratum elements are submitted first via `#extractBackground`, generally containing any blurring or background textures.

:::warning
Blurring, as handled through `GuiGraphicsExtractor#blurBeforeThisStratum`, can only be called once on any given frame. Attempting to submit a second blur will cause an exception to be thrown.
:::

The element stratum elements are submitted next via the `#extractRenderState` method, provided by being a `Renderable` subtype. This mainly submits widgets and labels, along with setting the hoverables to submit.

> Finally, the hoverable stratums submits things that hover over the previous elements, such as tooltips.

Reviewer: ```suggestion
Finally, the hoverable strata submit elements that hover over the previous elements, such as tooltips.
```

### Rendering the Container Screen

> The container screen uses all three strata to submit its elements. First, the background stratum submits the background texture by overriding `#extractBackground`. Then, the element stratum submits the widgets like before within `#extractContents`, followed by labels in `#extractLabels`. Finally, `AbstractContainerScreen` sets up the tooltip to be submitted during the hoverable stratums via `extractTooltip`

Reviewer: ```suggestion
The container screen uses all three strata to submit its elements. First, the background stratum submits the background texture by overriding `#extractBackground`. Then, the element stratum submits the widgets like before within `#extractContents`, followed by labels in `#extractLabels`. Finally, `AbstractContainerScreen` sets up the tooltip to be submitted during the hoverable strata via `extractTooltip`
```
